what-is-the-ic/3_canisters/index.md

```diff
@@ -4,16 +4,11 @@ card: /img/what-is-the-ic/canister.jpg
 cardImageFit: cover
 ---
 
-A smart contract is a computer program executed on a blockchain. A *canister*, or canister smart contract, is a bundle comprising a computer program and its data. Every canister is hosted on one subnet of the IC.
+A smart contract is a computer program executed on a blockchain. A canister is a smart contract that bundles a computer program and its data. Every canister is hosted on one subnet of the IC.
```

Comment:
```suggestion
A canister is a smart contract that bundles a computer program and its data. A smart contract is a computer program executed on a blockchain. Every canister is hosted on one subnet of the IC.
```
Feels weird to begin with 'smart contract' on a page that should talk about canisters

Reply:
Also feels weird to say that a canister is a smart contract without saying what a smart contract is ;) 
@THLO @dietersommer any opinion?